### PR TITLE
Add newsletter's subscription banner

### PR DESCRIPTION
**Pull request by quaxsze**
fix https://github.com/etalab/data.gouv.fr/issues/191

### CODE REVIEW

**Comment by quaxsze** [comment]
Le lien retenu  : `https://infolettres.etalab.gouv.fr/subscribe/rn7y93le1`
Le wording retenu pour l'instant : `Pour ne rien manquer, de l’actualité de data.gouv.fr et de l’open data, inscrivez-vous à notre infolettre.`

File: udata_gouvfr/settings.py [hunk 1]
```diff
@@ -13,3 +13,8 @@
 # api.gouv.fr
 APIGOUVFR_URL = 'https://api.gouv.fr/api/v1/apis'
 APIGOUVFR_ALLOW_OPENNESS = ['open', 'semi_open']
+
+# Newsletter's subscription page's link
+NEWSLETTER_SUBSCRIPTION_BANNER = False
+NEWSLETTER_LINK = ''
```
Comment (abulte) [inline udata_gouvfr/settings.py:19]: you can add those values as default https://github.com/etalab/udata-gouvfr/pull/512#issuecomment-733806815

**Review by abulte — COMMENTED** [review]
Screenshot? I'd like to see the look of it.

**Review by abulte — COMMENTED** [review]
We should call this `POST_BANNER`. The way it's designed, it has no specificity regarding a newsletter.

Also, we could show it at the top or bottom of the post detail page, but maybe later.

**Comment by quaxsze** [comment]
image

**Comment by abulte** [comment]
Thx for the screenshot. Either the link is missing or a style (underlined) is missing.

**Comment by quaxsze** [comment]
The link is there but it set the text is underlined only when hoverring

**Comment by abulte** [comment]
Okay, can you make it always underlined? It's much more discoverable and readable.

File: udata_gouvfr/theme/templates/post/list.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends 'post/list.html' %}
+
+{% set a_link = config['POST_BANNER_LINK'] %}
+{% set a_content = config['POST_BANNER_MESSAGE'] %}
+
+{% block alert_section %}
+{% if config['POST_BANNER_ACTIVATED'] %}
+<div class="container-fluid alert-info alert">
+    <div class="container">
```
Comment (abulte) [inline udata_gouvfr/theme/templates/post/list.html:9]: keep the role from here https://github.com/etalab/udata-gouvfr/pull/512/commits/8b7f83ca1d1e750e01006f17073def4979cd0bff#diff-a7db4438233ee9ea6df522d198260ba9dc57f978809ab6fd144399fda794daffL9

File: theme/less/gouvfr/alerts.less [hunk 3]
```diff
@@ -0,0 +1,12 @@
+@import "~bootstrap/less/mixins";
+@import "variables";
+
+/**
+ * Alert styles.
+ */
+
+.alert-info {
```
Comment (abulte) [inline theme/less/gouvfr/alerts.less:8]: use `.alert`, this is generic enough

**Review by abulte — APPROVED** [review]

**Pull request merged**